src/sdk/sql_cluster_router.h

```diff
@@ -119,6 +119,8 @@ class SQLClusterRouter : public SQLRouter {
     bool DropTable(const std::string& db, const std::string& table, hybridse::sdk::Status* status);
 
     bool ShowDB(std::vector<std::string>* dbs, hybridse::sdk::Status* status) override;
+    
+    std::vector<std::string> GetAllTables() override;
```

Comment:
pls write tests in both `cpp` & `python` side

Reply:
only write unittest in python currently